radio/src/gui/9x/menu_model_setup.cpp

```diff
@@ -679,7 +679,7 @@ void menuModelSetup(uint8_t event)
               break;
             case 1:
               CHECK_INCDEC_MODELVAR(event, g_model.ppmNCH, -2, 4);
```

Comment:
Yes, there is an ifdef, I missed that because it cover not only that statement but a large area